ui/src/app/ErrorInfo/ErrorInfo.tsx

```diff
@@ -1,38 +1,41 @@
 import React from 'react';
 import { Popover, Icon, Tooltip } from '@patternfly/react-core';
-import { TableComposable, Tr, Th, Td } from '@patternfly/react-table'
+import { TableComposable, Tr, Th, Td, Tbody } from '@patternfly/react-table';
 import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 import { DeploymentStepStatusData } from '@app/apis/types';
 
-import './ErrorInfo.css'
+import './ErrorInfo.css';
 
 interface IErrorInfoProps {
   stepStatusData: DeploymentStepStatusData;
 }
 
 export const ErrorInfoPopover = ({ stepStatusData }: IErrorInfoProps) => {
-
   const data = (
     <>
-    <TableComposable>
-        <Tr>
+      <TableComposable>
```

Comment:
Since this has one top level element we don't need to wrap it in `<> </>`.

Reply:
OK!